Resources/Private/JavaScript/Host/Containers/FullScreen/index.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {actions} from 'Host/Redux/';
+import {IconButton} from 'Host/Components/';
+import style from './style.css';
+import {immutableOperations} from 'Shared/Utilities/';
+
+const {$get} = immutableOperations;
+
+@connect(state => ({
+    isFullScreen: $get(state, 'ui.fullScreen.isFullScreen')
+}))
+export default class ContextBar extends Component {
```

Comment:
Looks like `ContextBar` got copied accidentally ;) - I would name the component `FullscreenToggler` though... But `Fullscreen` might be also okay... matter of taste I think :)